README.md

```diff
@@ -12,6 +12,8 @@ The [RAPIDS](https://rapids.ai) cuGraph library is a collection of graph analyti
 import cugraph
 
 # assuming that data has been loaded into a cuDF (using read_csv) Dataframe
+gdf = cudf.read_csv("graph_data.csv", names=["src", "dst"], dtype=["int32", "int32"] )
+
 # create a Graph using the source (src) and destination (dst) vertex pairs the GDF  
 G = cugraph.Graph()
 G.add_edge_list(gdf["src"], gdf["dst"])

```

Comment:
We may want to use the new 0.11 API here
```
    G = cugraph.DiGraph()
    G.from_cudf_edgelist(gdf, source='src', destination='dst')
```

Reply:
I'll update that plus add a note/link to the transition page